packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -920,7 +920,7 @@ export class List<T = any> extends BaseComponent<IListProps<T>, IListState<T>> i
       return;
     }
 
-    let surfaceRect = this._surfaceRect;
+    let surfaceRect = this._surfaceRect || { ...EMPTY_RECT };
```

Comment:
What's the purpose of this? Just curious

Reply:
`this._surfaceRect` can be `undefined` in _some_ cases such as snapshot tests where the viewport does not exist. In those cases we need to guard with default values else we risk throwing run-time exceptions when accessing those properties.

I felt this was safest compared with refactoring the existing logic throughout `List`. In my testing the only scenario impacted by this was snapshot tests.